lib/transaction/operation/congress_voting.go

```diff
@@ -13,9 +13,11 @@ type CongressVoting struct {
 		Start uint64 `json:"start"`
 		End   uint64 `json:"end"`
 	} `json:"voting"`
+	FundingAddress string        `json:"funding_address"`
```

Comment:
@outsideris CongressVoting and CongressVotingResult operations will be changed by this PR.